### ISSUE TITLE
The global Lua built-in error function randomly gets set to nil

### ISSUE THREAD

**Issue by Mayron** [AeroScripts/QuestieDev#1246]
## Bug description
This is a very strange bug but it affected some of my other AddOns today. Basically, as soon as you log in with just Questie enabled and spam the following macro:

```
/run print(error)
```

Eventually, maybe after 5 seconds it will stop printing the function memory address and will start printing `nil`. I'm guessing this is going to be a tricky one to fix!

## Screenshots
![image](https://user-images.githubusercontent.com/5854995/66339663-7feb9a80-e93b-11e9-860c-781ef4923f2b.png)

## Questie version
I am using the latest version for WoW Classic (on WoW Classic): **4.1.1**

**Comment by Mayron** [comment]
Is it perhaps caused by the `Questie\Modules\QuestieTracker.lua`? There are many of these lines:

```lua
result, error = pcall(frm.SetPoint, frm, unpack(Questie.db.char.TrackerLocation))
```

Looks like it's reassigning the global error variable to something else that might return `nil`. Plus, result is a global?

**Comment by Mayron** [comment]
Yep, that was the problem:

![image](https://user-images.githubusercontent.com/5854995/66340457-1bc9d600-e93d-11e9-9b25-9bcf89530017.png)

**Comment by Mayron** [comment]
There are also a lot of references to `result` as a global which gets set to true at some point. I fixed them and wanted to make a pull request but I have the classic version and there seems to be a lot of differences with my questie classic download compared to the master branch.

**Comment by BreakBB** [comment]
This has already been fixed with the latest release of v4.1.2. We had some bad code parts which lead to this issue

**Comment by Mayron** [comment]
Thanks @BreakBB, but I update my addons using the twitch client/curseforge which says the latest version is 4.1.1. Will 4.1.2 be released on curseforge any time soon? Or should I download it someplace else?

**Comment by BreakBB** [comment]
Yes we haven't released it on Twitch yet because we're still writing the changelog. Lots of changes included in v4.1.2

You can grab this version directly from Github. Have a look at [this install guide](https://github.com/AeroScripts/QuestieDev/wiki/FAQ%26Readme#basic-installation-instructions) to learn how.
